bridge/client/app/_components/ktb-notification-bar/ktb-notification-bar.component.scss

```diff
@@ -1,65 +1,11 @@
-@import '../../../variables';
-@import '~@dynatrace/barista-components/core/src/style/variables';
-
-.page-note {
-  z-index: 101;
+.notification-container {
+  pointer-events: none;
+  position: absolute;
```

Comment:
Idk maybe `position: fixed;` makes more sense if it may be provided in other components than root.